consoles/sshXtermVt.pm

```diff
@@ -62,6 +67,23 @@ sub activate {
     }
 }
 
+sub wait_for_ssh_port {
+    my ($self, $hostname, %args) = @_;
+    $args{timeout} //= 120;
+    $args{port}    //= 22;
+
+    bmwqemu::diag("Wait for SSH on host $hostname (timeout: $args{timeout})");
+
+    $args{timeout} = 1 unless ($args{timeout} > 0);
+    my $endtime = time() + $args{timeout};
+    while (time() < $endtime) {
+        my $sock = IO::Socket::INET->new(PeerAddr => $hostname, PeerPort => $args{port}, Proto => 'tcp', Timeout => 1);
+        return 1 if (defined $sock);
+        sleep 1;
+    }
+    return 0;
+}
+
```

Comment:
hey - we already have a similiar check here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/opensusebasetest.pm#L674

you might want to unify those

Reply:
That one is not working in an sshXtermVt context since you need to check before starting Xterm. Otherwise XtermVT just hangs.
So, both should be kept.

Reply:
ok but I would like to see how this looks like in real openQA tests, e.g. that this does not cause any hard to debug "incompletes"

Reply:
No `incompletes`:
```
[2019-12-10T17:18:06.510 CET] [debug] Connected to Xvnc - PID 26434
[2019-12-10T17:18:07.531 CET] [debug] Wait for SSH on host 192.168.0.15 (timeout: 120)
[2019-12-10T17:20:07.769 CET] [debug] 192.168.0.15 does not seems to have an active SSH server. Continuing anyway.

[2019-12-10T17:20:07.838 CET] [debug] led state 0 0 0 -261
[2019-12-10T17:20:07.852 CET] [debug] activate_console, console: root-console, type: ssh
[2019-12-10T17:20:07.853 CET] [debug] /var/lib/openqa/share/tests/opensuse/tests/jeos/prepare_firstboot.pm:34 called opensusebasetest::select_serial_terminal
[2019-12-10T17:20:07.853 CET] [debug] <<< testapi::assert_screen(mustmatch='password-prompt', timeout=30)
[2019-12-10T17:20:07.977 CET] [debug] no match: 59.9s, best candidate: password-prompt-20141126 (0.00)
[2019-12-10T17:20:08.860 CET] [debug] no change: 58.9s
```

And finally fail on test as usual.